src/components/Button/index.js

```diff
@@ -73,15 +78,19 @@ export default class Button extends Component {
             type,
             ariaHaspopup,
             id,
+            isLoading,
+            variant,
         } = this.props;
 
+        const isDisabled = this.isDisabled();
```

Comment:
It is fine, but here you don't need to create a method for this since it is not used in any other place.